plugin/files/airconsole/runtime.js

```diff
@@ -86,6 +88,7 @@ cr.plugins_.AirConsole = function(runtime)
       if (!device_data) {
         self.ac_leave_id = device_id;
         self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnDeviceLeft, self);
+		self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnAnyDeviceLeft, self);
```

Comment:
Great! Can you fix those things and then push the new changes with 1.5.0?! Saw you also implemented the highscores! 👍